roles/contiv/handlers/main.yml

```diff
@@ -0,0 +1,14 @@
+---
+- name: reload systemd
+  command: systemctl --system daemon-reload
+
+- name: restart netmaster
+  service: name=netmaster state=restarted
+  when: netmaster_started.changed == false
+
+- name: restart netplugin
+  service: name=netplugin state=restarted
+  when: netplugin_started.changed == false
```

Comment:
The convention used in all the contiv role files matches with "service: name=netplugin state=restarted".  Not clear when you say its a mix ...